src/peft/tuners/lora/layer.py

```diff
@@ -1958,7 +1958,11 @@ def _move_adapter_to_device_of_base_layer(self, adapter_name: str, device: Optio
                 adapter_layer[adapter_name] = adapter_layer[adapter_name].to(device)
 
     def get_param(self):
-        return getattr(self.base_layer, self.parameter_name)
+        if isinstance(self.base_layer, ParamWrapper):
+            param = getattr(self.get_base_layer(), self.parameter_name)
+        else:
+            param = getattr(self.base_layer, self.parameter_name)
+        return param
```

Comment:
Why the distinction, why not always use `self.get_base_layer()`? Let's also document the reason

Reply:
It was just defensive programming, but yeah, I don't think always calling `self.get_base_layer()` has any practical disadvantage.